test/rules/assertions/checkLineAlignment.js

```diff
@@ -917,6 +917,78 @@ export default {
       const fn = ( lorem, sit ) => {}
       `,
     },
+    {
+      code: `
+      /**
+       * Function description.
+       *
+       * @param  {string} lorem Description.
+       * @param  {int}    sit   Description multi words.
+       *
+       * @return {string}       Return description.
+       */
+      const fn = ( lorem, sit ) => {}
+      `,
+      errors: [
+        {
+          line: 2,
+          message: 'Expected JSDoc block lines to be aligned.',
+          type: 'Block',
+        },
+      ],
+      options: ['always', {
+        customSpacings: {
+          postDelimiter: 2,
+          postTag: 3,
+          postType: 2,
+        },
+      }],
+      output: `
+      /**
+       * Function description.
+       *
+       *  @param    {string}  lorem Description.
+       *  @param    {int}     sit   Description multi words.
+       *
+       *  @return   {string}        Return description.
+       */
+      const fn = ( lorem, sit ) => {}
+      `,
+    },
+    {
+      code: `
+      /**
+       * Function description.
+       *
+       *  @param   {string}  lorem Description.
+       * @param {int} sit Description multi words.
+       */
+      const fn = ( lorem, sit ) => {}
+      `,
+      errors: [
+        {
+          line: 6,
+          message: 'Expected JSDoc block lines to not be aligned.',
+          type: 'Block',
+        },
+      ],
+      options: ['never', {
+        customSpacings: {
+          postDelimiter: 2,
+          postTag: 3,
+          postType: 2,
+        },
+      }],
+      output: `
+      /**
+       * Function description.
+       *
+       *  @param   {string}  lorem Description.
+       *  @param   {int}  sit Description multi words.
+       */
+      const fn = ( lorem, sit ) => {}
+      `,
+    },
   ],
```

Comment:
How about at least one valid test, and for invalid, adding one test with an explicit `postName` set to 2+ (different from the default).

Reply:
Sure! https://github.com/gajus/eslint-plugin-jsdoc/pull/740/commits/3d626b0e51bace5e8bc02ca8a56075720c69c86c